stubs/commonmark/commonmark/inlines.pyi

```diff
@@ -50,14 +50,14 @@ class InlineParser:
     def removeDelimiter(self, delim: dict[str, Any]) -> None: ...
     @staticmethod
     def removeDelimitersBetween(bottom: dict[str, Any], top: dict[str, Any]) -> None: ...
-    def processEmphasis(self, stack_bottom) -> None: ...
+    def processEmphasis(self, stack_bottom: dict[str, Any]) -> None: ...
     def parseLinkTitle(self) -> str | None: ...
     def parseLinkDestination(self) -> str | None: ...
     def parseLinkLabel(self) -> int: ...
     def parseOpenBracket(self, block: Node) -> Literal[True]: ...
     def parseBang(self, block: Node) -> Literal[True]: ...
     def parseCloseBracket(self, block: Node) -> Literal[True]: ...
-    def addBracket(self, node, index, image) -> None: ...
+    def addBracket(self, node: Node, index: int, image: bool | None) -> None: ...
```

Comment:
Not sure if `None` is completely appropriate here. It technically works, but looking at the source, it's always used as a boolean, no `None` check, and only ever passed `True` or `False` in this method's usage.

But I also see that when a parameter is unused, the source defaults it to `None`, even boolean.

And @srittau already accepted those in https://github.com/python/typeshed/pull/13681

Reply:
That's why I left `None` because the internal logic doesn't prohibit it

maybe someone prefers not to pass this argument then it will work as `False`

Reply:
Well in this case if you don't pass an argument you'll get `TypeError: addBracket() missing 1 required positional argument: 'image'` 😉 
But semantic details aside, reviewing this PR made me realize the library this stubs is archived and deprecated. And now that you completed it, I think it's good for a sendoff. https://github.com/python/typeshed/issues/13725

Reply:
indeed, this can be fixed together with dict in PR before deletion